src/extra/AutoUI/Filters/PersistentFilters.tsx

```diff
@@ -44,27 +46,24 @@ const isQueryStringFilterRuleset = (
 	!!rule?.length &&
 	rule?.every(isListQueryStringFilterRule);
 
-const listFilterQuery = (schema: JSONSchema, rules: JSONSchema[]) => {
-	const queryStringFilters = rules.map((filter) => {
-		// TODO: The typings need to be fixed in rendition.
-		const flatSchema = flattenSchema(schema);
-		const flattenFilter = flattenSchema(filter);
-		const signatures = decodeFilter(
-			flatSchema,
-			flattenFilter,
-		) as FilterSignature[];
+export const listFilterQuery = (filters: JSONSchema[]) => {
+	const queryStringFilters = filters.map((filter) => {
+		const signatures = getSignatures(filter) as FilterSignature[];
```

Comment:
Why is this cast necessary? Looks like `FiltersSignature[]` is already the return type of `getSignatures`

Reply:
nice catch! This was one of the multiple tries that I've done and I forgot to remove it